src/ert/config/gen_kw_config.py

```diff
@@ -383,6 +383,11 @@ def load_parameters(
     ) -> npt.NDArray[np.float64]:
         return ensemble.load_parameters(self.name, realizations)["values"].values.T
 
+    def load_parameters_for_update(
```

Comment:
Not sure how this makes sense at the moment? 
1) There is the `update` flag on each field, surf and genkw parameter and it is not used here.
2) I'd prefer that there would be a local_ensemble.load_parameters_for_update(...) at least if would give more sense to me.